dropwizard-jsonapi/src/main/java/io/rtr/jsonapi/filter/JsonApiResponseFilter.java

```diff
@@ -115,20 +115,23 @@ private ResourceObjectImpl buildEntity(final Mapping mapping, final Object entit
 				if (id != null) {
 					final String uri = template.createURI(id);
 					final String linkSelfUri = uriInfo.getBaseUri().resolve(uri.substring(1)).toString();
+					final List<Object> included = resolveIncludes(resource, entity, id, key, m);
```

Comment:
yes, we need to get the Id in all cases, but we won't populate the include section unless asked to